web-registry/src/utils/mui/sxToArray.ts

```diff
@@ -0,0 +1,6 @@
+import { SxProps, Theme } from '@mui/material';
+
+type SxPropsType = SxProps<Theme>;
+
+export const sxToArray = (sx?: SxPropsType): any[] =>
```

Comment:
```suggestion
export const sxToArray = (sx?: SxPropsType): SxPropsType[] =>
```

Reply:
Not sure why I used an any here 🤔 , thanks for catching it.

Reply:
I remember now, using `SxProps<Theme>[]` as the return type generates a TS error that I could not figure out here: https://github.com/regen-network/regen-web/pull/1738/files#diff-0a09b7002e714217699709642eed0e4dd157fcb0164a98d49b4732f3c7d0e2abR42
![image](https://user-images.githubusercontent.com/530644/214247816-2be6193c-7307-4ceb-87c1-ced2adb06df4.png)
`Index signature for type 'string' is missing in type 'readonly (boolean | SystemStyleObject<Theme> | ((theme: Theme) => SystemStyleObject<Theme>))[]'.ts(2322)`
Let me know if you have an idea to solve this one.